packages/blockchain-wallet-v4/src/network/api/nfts/index.ts

```diff
@@ -69,6 +70,15 @@ export default ({ apiUrl, get, openSeaApi, post }) => {
     })
   }
 
+  const getNftOwnerAssets = (defaultEthAddr?: string, cursor?: string): OwnerNftBalance => {
+    return get({
+      contentType: 'application/json',
+      endPoint: `/account_assets/${defaultEthAddr}/${cursor || ''}`,
```

Comment:
since defaultEthAddr and cursor are optional should you check them first? Otherwise you might be calling `/account_assets/undefined/undefined`